### DISCUSSION TITLE
http://localhost:3000/datasets/detail?pageType=datasets&name=H2O-Phase-Diagram&id=1

### DISCUSSION THREAD

**Discussion by giscus** [yanyumei/giscus-test#11] [Announcements]
# http://localhost:3000/datasets/detail?pageType=datasets&name=H2O-Phase-Diagram&id=1

http://localhost:3000/datasets/detail?pageType=datasets&name=H2O-Phase-Diagram&id=1

**Comment by smallplum** [comment]
你好👋

**Comment by smallplum** [comment]
好啊

**Reply by smallplum** [reply]
谢谢
